crates/bevy_log/src/lib.rs

```diff
@@ -134,7 +134,70 @@ pub(crate) struct FlushGuard(SyncCell<tracing_chrome::FlushGuard>);
 ///         .run();
 /// }
 /// ```
+/// # Example Setup
 ///
+/// For a quick setup that enables all first-party logging, while not showing any of your dependencies' log data
+/// you can configure the plugin as below.
+///
+/// ```no_run
+/// # use bevy_app::{App, NoopPluginGroup as DefaultPlugins, PluginGroup};
+/// # use bevy_log::*;
+/// App::new()
+///     .add_plugins(DefaultPlugins.set(LogPlugin {
+///         filter: "warn,my_crate=trace".to_string(), //specific filters
+///         level: Level::TRACE,//Change this to be globally change levels
+///         ..Default::default()
+///         }))
+///     .run();
+/// ```
+/// The filter (in this case an `EnvFilter`) chooses whether to print the log, the most specific filters apply
+/// lets start with an example `filter: "warn".to_string()` will only print logs with level `warn` level or greater
+/// From here we can change to `filter: "warn,my_crate=trace".to_string()` logs will only print at warn unless it's in `mycrate`
+/// which will print at `trace` level `my_crate=trace` as that clause is more specific
```

Comment:
```suggestion
/// The filter (in this case an `EnvFilter`) chooses whether to print the log. The most specific filters apply with higher priority.
/// Let's start with an example: `filter: "warn".to_string()` will only print logs with level `warn` level or greater.
/// From here, we can change to `filter: "warn,my_crate=trace".to_string()`. Logs will print at level `warn` unless it's in `mycrate`,
/// which will instead print at `trace` level because `my_crate=trace` is more specific.
```